cmake/platform.cmake

```diff
@@ -220,7 +220,7 @@ elseif(UNIX OR MINGW)
              endif()
              # For native compilation tune for the host processor
              if (CMAKE_SYSTEM_PROCESSOR STREQUAL CMAKE_HOST_SYSTEM_PROCESSOR)
-                 append(DEF_ARCH_OPT_FLAGS "-mcpu=native")
+                 append(DEF_ARCH_OPT_FLAGS "-march=armv8-a")
```

Comment:
Would `generic` be a better option rather than putting a specific isa?

Reply:
`armv8-a` is a minimal for aarm64.